### PR TITLE
Fixed error message css + email warning config option

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Fixed the styling of the error message:

error

fixed

## How To Test This

1. Install the CRM without setting up SMTP settings
2. View the error message in all 4 themes.
3. Check that the new checkbox field email warning notification is visible in email settings.
4. Check that the user can disable email warnings.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Mac-Rae** [comment]
Assessed :+1:

**Comment by horus68** [comment]
Don't like it... It will turn out this "tiny alert" in a huge big "alert" not required for those that do not use email functions with the CRM. Maybe an "Hide email alert message" somewhere?

**Comment by Dillon-Brown** [comment]
Hi @horus68, this was actually just a quick fix for a regression issue but I agree that we should probably have some way of hiding these. I'll push up another commit that adds a checkbox for this in email settings.

**Comment by cameronblaikie** [comment]
Assessed :+1:

File: modules/Administration/Administration.php [hunk 1]
```diff
@@ -81,6 +81,9 @@ public function checkSmtpError($displayWarning = true)
         $smtp_error = false;
         $this->retrieveSettings();
 
+        if (!$sugar_config['email_warning_notifications']) {
```
Comment (gymad) [inline modules/Administration/Administration.php]: Does it shows up a php notice in log after the upgrade?

**Pull request merged**